css/home.css

```diff
@@ -0,0 +1,968 @@
+}
+
+.main-info .post-title {
+  font-family: "Lato", sans-serif;
+  font-size: 16px;
+  color: #222;
+  border: 4px solid #fff;
+  padding: 10px 15px;
+  background-color: #f7f7f7;
+  max-width: 90%;
+  margin: 4% 2% 0 3%;
+  opacity: 0.9;
+  z-index: 80;
+}
+
+.main-info .dates {
+  font-family: "Cocogoose", sans-serif;
+  font-size: 1em;
+  color: #272a31;
+  font-weight: 900;
+  line-height: 50px;
+  width: 90%;
+  transform: scale(1.4);
+  margin-left: 20%;
+}
+
+.main-info .venue {
+  font-family: "Lato", sans-serif;
+  font-size: 14px;
+  color: #555;
+  font-weight: 600;
+  margin-top: -1.7em;
+  margin-left: 10px;
+  z-index: 1000;
+}
+
+.main-info .venue:hover {
+  cursor: pointer;
+  color: #ec5242;
+}
+
+section .separator {
+  width: 10%;
+  align-self: center;
+  background-color: #e74c33;
+  height: 2px;
+  margin-block: -1em 0.7em;
+}
+
+.main-program {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1em;
+  position: relative;
+  background-image: url("../assets/jpg/dark-gray-sand.jpg");
+  background-repeat: no-repeat;
+  background-attachment: scroll;
+  background-size: cover;
+  color: #f8f8f8;
+  padding-block: 1.2em;
+}
+
+.main-program .overlay {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background: #ce90ce;
+}
+
+.main-program .section-title {
+  font-family: "Lato", sans-serif;
+  font-size: 18px;
+  color: white;
+}
+
+.main-program .articles-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+}
+
+.main-program .program-item {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 25px;
+  align-items: center;
+  padding: 12px 8px;
+  margin-inline: 5%;
+  width: 90%;
+  background: #2b2929;
+  border: 2px solid #3c3f46;
+  opacity: 0.9;
+}
+
+.main-program .program-item .icon {
+  font-size: 1.7em;
+}
+
+.main-program .program-item .activity {
+  color: #ec5242;
+  font-size: 18px;
+  width: 15ch;
+  font-weight: bold;
+  z-index: 100;
+}
+
+.main-program .program-item .description {
+  color: #fff;
+  font-size: 16px;
+  text-align: left;
+}
+
+.main-program .see-program {
+  display: none;
+}
+
+.main-program .enroll-btn {
+  display: block;
+  cursor: pointer;
+  padding: 20px;
+  width: 60%;
+  text-align: center;
+  background: #e74c33;
+  color: #fff;
+  font-size: 1.2em;
+  margin-block: 1.3em 0.8em;
+  border: 1px solid #272a31;
+  font-weight: 400;
+}
+
+.instructors {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1em;
+  margin-inline: 15px;
+  background-color: #f8f8f8;
+  padding-bottom: 1em;
+}
+
+.instructors .section-title {
+  font-family: "Lato", sans-serif;
+  font-size: 18px;
+  color: #272a31;
+  margin-top: 2em;
+}
+
+.instructors .instructors-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1em;
+}
+
+.instructors .instructor {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 7px;
+}
```

Comment:
- [ ] In the instructor's section, the article's photos are aligned in the original design. But there is a misalignment in your design. Kindly make changes here. :clap: 
- [ ] In addition, the font for the description text is not the same as the original design. Please make sureto use the original typography. 

![image](https://user-images.githubusercontent.com/64894927/132559951-7a6884b8-7037-4df5-8598-0c5090e9ceec.png)

![image](https://user-images.githubusercontent.com/64894927/132559191-82c9d4f0-a28a-45db-add8-140cbdc9c17d.png)

Reply:
Sorry, I had to be absent from Thursday to Saturday last week. So I'm repeating the week after having used only a single attempt.
Those changes are done now.